qutebrowser/misc/utilcmds.py

```diff
@@ -247,3 +247,34 @@ def log_capacity(capacity: int):
         raise cmdexc.CommandError("Can't set a negative log capacity!")
     else:
         log.ram_handler.change_log_capacity(capacity)
+
+
+@cmdutils.register(debug=True, name='debug-log-level')
+@cmdutils.argument('level', choices=[level.lower()
+                    for level in log.LOG_LEVELS])
+def debug_log_level(level: str):
+    """Change the log level for console logging.
+
+    Args:
+       level: log level for console log.
+    """
+    try:
+        log.console_handler.setLevel(log.LOG_LEVELS[level.upper()])
+    except debug_log_level.Error as e:
```

Comment:
Could you explain why you're doing this and what you think it's doing? `debug_log_level` is a function, so `debug_log_level.Error` doesn't exist - so I'm not sure what your intention was here.